### PR TITLE
Add an initial PR template to the site

### PR DESCRIPTION
**Pull request by tomas-stefano**
This PR will solve to standardise our PR process.

Paired with @willhall88 

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1741)

### CODE REVIEW

**Comment by willhall88** [comment]
👍

**Comment by margOnline** [comment]
👍

**Comment by margOnline** [comment]
Before this is merged, I think it should be viewed by most if not all of the dev team - wdyt?

**Comment by Tr4pSt3R** [comment]
👍 👍

**Comment by Tr4pSt3R** [comment]
@tomas-stefano: @margOnline left a comment above 👆

**Comment by tomas-stefano** [comment]
Sorry, I didn't see your comments @margOnline

**Comment by willhall88** [comment]
The template can be updated once the whole dev team has used it and have feedback so not a bad thing for it to be merged

**Comment by tomas-stefano** [comment]
My idea was more like continuous improvement ... try to get one first version first (that I know will not be the perfect version) and then we change along during the process.

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Before / after images of any visual / ui updates.
+
+**QA / Testing**
+Testing notes / guidelines if necessary.
+
+** Checklist **
+
+Make sure that when your PR is good to be reviewed you check the below checks:
+
++ [ ] https://{$QA_OR_PREVIEW_OR_UAT_TEST_LINK_FOR_THIS_PR} /
```
Comment (johnplayer1982) [inline .github/PULL_REQUEST_TEMPLATE]: Not sure this will be required, as the changes wont be on a test URL until the PR has been approved?

Comment (tomas-stefano) [inline .github/PULL_REQUEST_TEMPLATE]: Good call. I will remove it.

**Pull request merged**